examples/get-file-contents.js

```diff
@@ -0,0 +1,112 @@
+import {
+    PrivateKey,
+    FileContentsQuery,
+    Hbar,
+    FileCreateTransaction,
+    FileDeleteTransaction,
+    Client,
+    AccountId,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * How to get file contents
+ *
+ */
+
+async function main() {
+    /**
+     * Step 0:
+     * Create and configure the SDK Client
+     */
+    if (
+        process.env.OPERATOR_ID == null ||
+        process.env.OPERATOR_KEY == null ||
+        process.env.HEDERA_NETWORK == null
+    ) {
+        throw new Error(
+            "Environment variables OPERATOR_ID, HEDERA_NETWORK, and OPERATOR_KEY are required.",
+        );
+    }
+
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+
+    // Create a client for the local Hedera network
+    const client = Client.forNetwork({
+        [process.env.HEDERA_NETWORK]: new AccountId(3),
```

Comment:
As the HEDERA_NETWORK environment variable is typically set to **local-node**, **previewnet**, **testnet**, or **mainnet**, the current configuration isn't working because forNetwork expects a key
pair with the network's IP address and a consensus node ID, as shown in the other examples. It might be working in your environment because you've set HEDERA_NETWORK to the local IP address, but that's not the standard process we follow. Please update this, or refer to the other methods for setting up the local network.

Reply:
sorted